src/PublicLab.Editor.js

```diff
@@ -42,11 +42,7 @@ PL.Editor = Class.extend({
 
       });
 
-      if (valid_modules == required_modules) {
-
-        $('.ple-publish').removeClass('disabled');
-
-      }
+      $('.ple-publish').prop('disabled', !(valid_modules == required_modules));
```

Comment:
Adhering to the "triple equals" convention, can you change this to `$('.ple-publish').prop('disabled', (valid_modules !== required_modules));` instead?

Reply:
Done.